code/datums/gamemode/factions/blob.dm

```diff
@@ -69,12 +70,15 @@
 	if(outbreak_announcement && world.time >= outbreak_announcement && detect_overminds()) //Must be alive to advance.
 		outbreak_announcement = 0
 		stage(FACTION_ACTIVE)
-	if (declared && 0.20*blobwincount)
+	if (declared && 0.20*blobwincount <= blobs.len && defcon > 3)
```

Comment:
why are we not just doing this with stages?

this is the whole point of stages

Reply:
done